templates/arm/qemu/mods.conf

```diff
@@ -114,6 +114,7 @@ configuration conf {
 	include embox.cmd.net.tftp
 	include embox.cmd.net.snmpd
 	include embox.cmd.net.ntpdate
+	include embox.cmd.net.ntpd
```

Comment:
It's working? Or added just to make sure t compiled?

Reply:
It's only for the checking compilation